site/_docs/reference.md

```diff
@@ -2790,6 +2790,7 @@ In the following:
 | b f s | LENGTH(string)                             | Equivalent to `CHAR_LENGTH(string)`
 | h s | LEVENSHTEIN(string1, string2)                | Returns the Levenshtein distance between *string1* and *string2*
 | b | LOG(numeric1 [, numeric2 ])                    | Returns the logarithm of *numeric1* to base *numeric2*, or base e if *numeric2* is not present
+| m s | LOG(numeric1 [, numeric2 ])                  | Returns the logarithm of *numeric1* to base *numeric2*, or base e if *numeric2* is not present, or null if *numeric1* is 0
```

Comment:
if *numeric1* is 0 or negative ?

Reply:
Thank you for your reply. The negative you mentioned is a negative number.

Reply:
You are correct, I should have added a test for negative numbers, I will refine it later

Reply:
done